scripts/publish_image

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# set readable variables
+export JAVA_VERSION_TEMPLATE="$1"
+export NODE_VERSION_TEMPLATE="$2"
+export MBT_VERSION="$3"
+export IMAGE_PREFIX="$4"
+
+# set jvm and node version
+if [ "$JAVA_VERSION_TEMPLATE" = "11" ]
```

Comment:
similar to above comment about possible duplication in the if/else branches.
Could this be simplified?

Reply:
yes, the if/else can be simplified to :

export JAVA_VERSION="-java$JAVA_VERSION_TEMPLATE"
export NODE_VERSION="-node$NODE_VERSION_TEMPLATE"

Reply:
I will verify the above modification in my testing env and commit the code asap